actions/wait-for-deployment/entrypoint.sh

```diff
@@ -2,6 +2,21 @@
 
 set -eu
 
+#----------- [API Deployment Only] Wait for Integration API Pod to spin back up with correct version --------------
```

Comment:
```suggestion
# [API Deployment Only] Wait for Integration API Pod to spin back up with correct version
```

The comment line lengths are different with the separators, so I opted to remove them.